LayoutTests/fast/selectors/internal-pseudos.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Selectors: internal pseudos are not web exposed</title>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<script src="../../imported/w3c/web-platform-tests/css/support/parsing-testcommon.js"></script>
```

Comment:
I’m a little worried about tying tests outside WPT to the structure of WPT. Maybe we could contribute this to WPT. No platform or engine would want these pseudo selectors to be exposed, whether they exited or not, so it would be OK for others.

Reply:
That sounds reasonable.